tests/distro-regression/tests/bundles/kinetic-zed.yaml

```diff
@@ -7,6 +7,13 @@ variables:
   # Set retrofit-series to focal to work-around the following bug:
   # https://bugs.launchpad.net/snap-octavia-diskimage-retrofit/+bug/1995130
   retrofit-series: &retrofit-series focal
+  openstack-channel: &openstack-channel zed/edge
+  ceph-channel: &ceph-channel quincy/edge
+  ovn-channel: &ovn-channel 23.03/edge
+  mysql-channel: &mysql-channel latest/edge
+  rabbitmq-channel: &rabbitmq-channel latest/edge
+  memcached-channel: &memcached-channel latest/edge
+  vault-channel: &vault-channel latest/edge
```

Comment:
LGTM, some of the misc charms haven't gained kinetic support in their stable tracks